Added subclasses of Select25 that use properties files for localization.

matthewgeer opened the pull request.

matthewgeer committed:
Added subclasses of Select25 that use properties files for localization.

ivaynberg commented:
I dont think we need subclasses or the interface, move the createDictionary into the abstract class with the default implementation there and that should be good enough, no?

matthewgeer committed:
Rolled localization features into base Select25 controls rather than a subclass.

Created an overrideable addDictionaryToSettings method in case someone wanted to use the original TypeScript system.

matthewgeer commented:
Added a second commit rolling the features into the base class; I also added an addDictionaryToSettings method that can be overriden with a no-op in case someone wanted to use the original TypeScript localization system.
